danuluma/andela-project
# #160759291 configured badges

#### What does this PR do?
Configures the badges so that they show the build status and tests coverage

#### Description of Task to be completed?
Added scripts for tests to automatically run when a commit is pushed to GitHub

#### How should this be manually tested?
Navigate to the readme on the bottom of any branch in this repo.
You should see the build and test coverage badges

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
 #160759291

#### Screenshots (if appropriate)
#### Questions:
